public/configs/someMarketData.json

```diff
@@ -0,0 +1,1890 @@
+      "delayBlocks": 3600
+    },
+    "initialDeposit": {
+      "denom": "adydx",
+      "amount": "10000000000000000000000"
+    },
+    "meta": {
+      "assetName": "DADDY TATE",
+      "referencePrice": 0.1152154052164651,
+      "cmcId": -1
+    }
+  },
+  "LAIKA": {
+    "title": "Add LAIKA-USD perpetual market",
+    "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a LAIKA-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
+    "params": {
+      "id": 667,
+      "ticker": "LAIKA-USD",
+      "marketType": "PERPETUAL_MARKET_TYPE_ISOLATED",
+      "priceExponent": -13,
+      "minPriceChange": 800,
+      "minExchanges": 1,
+      "exchangeConfigJson": [
+        {
+          "exchangeName": "Raydium",
+          "ticker": "LAIKA-SOL-3eoSzSxo5XJiFbhDk2AMbRzqXMjntbp22fsjdoswmNLg-6-CwRJpNPtJBUTnYNVf6AsVDSWViffpuV7eFuoGKDKSaDo-9-DJyENBeqKAR8xhm2azRJYNQaGXPaWb6dgP5uUTJWvpJ-AJT2iW7P7G3m96vrL3ZRUU4kPnJCciSEjkfik1c9ZKYm",
+          "adjustByMarket": "SOL-USD",
+          "invert": false
+        }
+      ],
+      "liquidityTier": 2,
+      "atomicResolution": -2,
+      "quantumConversionExponent": -9,
+      "defaultFundingPpm": 0,
+      "stepBaseQuantums": 1000000,
+      "subticksPerTick": 1000000,
+      "delayBlocks": 3600
+    },
+    "initialDeposit": {
+      "denom": "adydx",
+      "amount": "10000000000000000000000"
+    },
+    "meta": {
+      "assetName": "Laika The Soldog",
+      "referencePrice": 0.0001221847556166,
+      "cmcId": -1
+    }
+  },
+  "NGMI": {
+    "title": "Add NGMI-USD perpetual market",
+    "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a NGMI-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
+    "params": {
+      "id": 668,
+      "ticker": "NGMI-USD",
+      "marketType": "PERPETUAL_MARKET_TYPE_ISOLATED",
+      "priceExponent": -10,
+      "minPriceChange": 800,
+      "minExchanges": 1,
+      "exchangeConfigJson": [
+        {
+          "exchangeName": "Raydium",
+          "ticker": "NGMI-SOL-12gsGzqnqJx8JpWz3W6PGn7iL8MVU787xrWsAESC4udk-9-D1xPodmwjnsDuyXi6Dwapphs2w7cxU8MoyFSykYcnFFx-9-DhpMtmWVExvRyCVYw1Qp3JgcUzicLeea2UBXs7Rm5STM-xdMgrDfMUDwBgJU1s1TjgFAGipXrEPwjgNJRcvdwoRe",
+          "adjustByMarket": "SOL-USD",
+          "invert": false
+        }
+      ],
+      "liquidityTier": 2,
+      "atomicResolution": -5,
+      "quantumConversionExponent": -9,
+      "defaultFundingPpm": 0,
+      "stepBaseQuantums": 1000000,
+      "subticksPerTick": 1000000,
+      "delayBlocks": 3600
+    },
+    "initialDeposit": {
+      "denom": "adydx",
+      "amount": "10000000000000000000000"
+    },
+    "meta": {
+      "assetName": "NGMI",
+      "referencePrice": 0.1151970509576157,
+      "cmcId": -1
+    }
+  },
+  "Q": {
```

Comment:
Looks like this pair also has v low liquidity ([dexscreener](https://dexscreener.com/solana/4qpjcwzzyofpeqf1e9x8vkajzuzpcdcbu2makn42bxzc))

Reply:
Removed